common/views/components/CheckboxRadio/CheckboxRadio.tsx

```diff
@@ -59,6 +64,15 @@ const CheckboxRadioInput = styled.input.attrs<{ $type: string }>(props => ({
     border-width: 2px;
   }
 
+  &[disabled] {
+    & ~ ${CheckboxRadioBox} {
+      color: pink;
```

Comment:
💅

Reply:
Oops. But it _is_ Wednesday.

Reply:
I removed the changes, we can address those again when we actually add a single-checkbox filter.